application/app/controllers/dataverse/collections_controller.rb

```diff
@@ -7,8 +7,9 @@ def show
     @service = Dataverse::CollectionService.new(@dataverse_url)
     begin
       @page = params[:page] ? params[:page].to_i : 1
+      search_query = params[:q].present? ? ActionView::Base.full_sanitizer.sanitize(params[:q]) : nil
```

Comment:
As it is use several times in this method and for clarity, maybe better to:
`collection_id = params[:id]`

Reply:
done